entrypoint.sh

```diff
@@ -24,14 +24,14 @@ echo "/home/node/.pm2/logs/*.log /home/node/app/cli/build/logs/*.log {
 
 # Pull latest versions of the CLI and GUI
 
-git clone https://gitlab.com/shardeum/validator/cli.git
+git clone -b sys-129-point-dashboard-installer-to-github-repos https://github.com/shardeum/validator-cli.git cli
```

Comment:
Will we want to remove this `-b` argument before merging?

Reply:
yes! this was so QA could test the install